### PR TITLE
Add Getting Started Guide

### PR DESCRIPTION
**Pull request by HotSushi**
## Summary
This PR adds a getting started guide to the docsite. Its a quickstart tutorial to get something up and running quickly.

Language follows the ["Tutorial Language"](https://diataxis.fr/tutorials/)

## Changes
- [X] Documentation

For all the boxes checked, please include additional details of the changes made in this pull request.  

## Testing Done

- [X] Some other form of testing like staging or soak time in production. Please explain.
`npm run start` and captured snapshot image here:

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+sidebar_position: 2
+---
+# OpenHouse on Spark & HDFS
+
+In this guide, we will quickly set up a running environment and experiment with some simple SQL commands. Our
+environment will include all the core OpenHouse services such as [Catalog Service](./intro.md#catalog-service),
+[House Table service](./intro.md#house-table-service) and [others](./intro.md#control-plane-for-tables),
+[a Spark 3.1 engine](https://spark.apache.org/releases/spark-release-3-1-1.html) and
+also [HDFS namenode and datanode](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#NameNode+and+DataNodes).
+By the end of this walkthrough, we will have made some tables on OpenHouse, put data in them, retrieved the data
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:18]:
```suggestion
By the end of this walkthrough, we will have created some tables on OpenHouse, inserted data in them, and queried data.
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+and checked HDFS namenodes to see the data written.
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:19]:
```suggestion
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+### Get environment ready
+First, clone [OpenHouse github repository](https://github.com/linkedin/openhouse) and
+run `./gradlew build` command at the root directory. After the command succeeds you should see `BUILD SUCCESSFUL`
+message.
+
+```shell
+openhouse$main>  ./gradlew build
+```
+
+Then, go to the location `infra/recipes/docker-compose/oh-hadoop-spark` and execute `docker compose up` command.
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:38]:
```suggestion
Execute `docker compose -f infra/recipes/docker-compose/oh-hadoop-spark/docker-compose.yml up -d --build` command to bring up docker container for openhouse services, spark and HDFS.
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+# change directory
+openhouse$main>  cd infra/recipes/docker-compose/oh-hadoop-spark
+
+# spin up containers
+oh-hadoop-spark$main>  docker compose up –d
+[+] Running
+✔ Container local.spark-master            Running
+✔ Container local.openhouse-housetables   Running
+...
+```
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:50]:
```suggestion
```

File: docs/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+Voila! Just like that, we have all the core OpenHouse services up alongside Spark and Hadoop.
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:51]:
```suggestion
```

File: docs/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+### Run SQL commands
+Let us  execute some basic SQL commands to create table, add data and retrieve the data.
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:54]:
```suggestion
Let us execute some basic SQL commands to create table, add data and query data.
```

File: docs/getting_started.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+--conf spark.sql.catalog.openhouse=org.apache.iceberg.spark.SparkCatalog   \
+--conf spark.sql.catalog.openhouse.catalog-impl=com.linkedin.openhouse.spark.OpenHouseCatalog     \
+--conf spark.sql.catalog.openhouse.metrics-reporter-impl=com.linkedin.openhouse.javaclient.OpenHouseMetricsReporter    \
+--conf spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080   \
+--conf spark.sql.catalog.openhouse.auth-token=$(cat /var/config/openhouse.token) \
+--conf spark.sql.catalog.openhouse.cluster=LocalHadoopCluster
+```
+:::note
+the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080` points to the docker container
+running the [REST-based catalog service](./intro.md#catalog-service).
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:72]:
```suggestion
running the [OpenHouse Catalog Service](./intro.md#catalog-service).
```

File: docs/getting_started.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+First, we should log out of the container if we are on Spark driver and log in to the HDFS name node.
+
+```
+oh-hadoop-spark$main>  docker exec -it local.namenode bash
+
+root@aa91a7bc8575:/# hdfs dfs -ls -R /data/openhouse/
+```
+This directory has all the OpenHouse tables that were made earlier. You should find the Iceberg metadata
+files (`metadata.json`, `/metadata`) and the datafiles (in `/data`) that were created for the data we added
+in the previous section.
```
Comment (sumedhsakdeo) [inline docs/getting_started.md:131]:
```suggestion
```
TMI

**Review by sumedhsakdeo — CHANGES_REQUESTED** [review]
This is good, we should approach this user guide with a minimal set of commands 

1. build openhouse
2. run openhouse+spark+hdfs in docker
3. test create table, insert into, select from, enable sharing, and grant select

To avoid duplication in the documentation here and what we have in the [SETUP](https://github.com/linkedin/openhouse/blob/main/SETUP.md) guide. We could provide link the users to the SETUP guide.

Wdyt?

**Comment by HotSushi** [comment]
> This is good, we should approach this user guide with a minimal set of commands
> 
> 1. build openhouse
> 2. run openhouse+spark+hdfs in docker
> 3. test create table, insert into, select from, enable sharing, and grant select
> 
> To avoid duplication in the documentation here and what we have in the [SETUP](https://github.com/linkedin/openhouse/blob/main/SETUP.md) guide. We could provide link the users to the SETUP guide.
> 
> Wdyt?

Added a link to the source of truth ie: SETUP guide. 

Agree on keeping it minimal and practical!

**Review by sumedhsakdeo — APPROVED** [review]
Nice!

**Pull request merged**